lavalink/node.py

```diff
@@ -225,8 +227,9 @@ async def connect(self, timeout=None, secured=False):
         asyncio.TimeoutError
             If the websocket failed to connect after the given time.
         """
+        self.secured = secured
```

Comment:
I feel like we just shouldn't have the `secured` argument in this method. Especially that it kinda defeats the point of having the `secured` attribute if it always gets overridden on connect.

Reply:
Done